assets/onboarding/data/actions.js

```diff
@@ -37,9 +39,7 @@ export function* fetchSetupWizardData() {
 		const data = yield fetchFromAPI( {
 			path: API_BASE_PATH,
 		} );
-		yield successFetch( {
-			...data,
-		} );
+		yield successFetch( normalizeSetupWizardData( data ) );
```

Comment:
Should this be done in a reducer instead? Seems like a better fit for transforming data.

Reply:
I usually like to do this type of thing as early as possible. In the action, in this case. Because if we want to get the same action in other reducers, or create a middleware to handle these data, this is ready. Make sense to you?

Reply:
Yeah, I can accept this thinking too :)